pytorch_lightning/accelerators/ddp_accelerator.py

```diff
@@ -144,8 +144,9 @@ def train(self):
         model = self.trainer.model
 
         results = self.ddp_train(process_idx=self.task_idx, model=model)
-        if 'WORLD_SIZE' in os.environ:
-            del os.environ['WORLD_SIZE']
+        # TODO: try this skip
+        # if 'WORLD_SIZE' in os.environ:
+        #     del os.environ['WORLD_SIZE']
```

Comment:
@awaelchli is seems there is no restore anywhere else... so how is it possible it passes on master now?

Reply:
any idea why the env deletion was added here? especially after training...